gf-webapp-admin/src/component/rooms/room/slider/slides/addSlide/AddSlide.module.scss

```diff
@@ -25,6 +25,8 @@
   }
 
   button {
```

Comment:
you already have button in library. Why do u create another style here?

Reply:
Done